lib/openxml/has_properties.rb

```diff
@@ -42,7 +42,7 @@ def property(name, as: nil, klass: nil)
         class_eval <<-CODE, __FILE__, __LINE__ + 1
         def #{name}(*args)
           if instance_variable_get("@#{name}").nil?
-            group_index = #{@current_group}
+            group_index = #{@current_group.inspect}
```

Comment:
Was this a bug that snuck in through a previous PR — that you noticed when integration testing this?

Reply:
I believe so. If `@current_group` is `nil`, we want to output "nil" instead of nothing, since the whole thing here is string interpolation.